### PR TITLE
Updated readme to reflect mint support

### PR DESCRIPTION
**Pull request by acecilia**
Solves https://github.com/JamitLabs/Accio/issues/46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,14 @@ To **update** it to the latest version, run this instead:
 brew upgrade accio
 ```
 
+### Via [Mint](https://github.com/yonaskolb/Mint):
+
+To **install** the latest version of Accio, run this command:
```
Comment (fredpi) [inline README.md]: Would you mind adjusting this line so it gets clear the `install` command also updates Accio if it's already installed? That would be more in-line with the Homebrew installation instructions 😉

Maybe change it to `To **install** Accio or **update** to the latest version of Accio, run this command:`

**Review by fredpi — CHANGES_REQUESTED** [review]
Thanks for this PR 🚀 Just one little change, if you're okay with it ;)

File: README.md (hunk 1 above)
Comment (acecilia) [reply README.md]: Done :)

**Review by fredpi — APPROVED** [review]

**Comment by fredpi** [comment]
Looks good, will merge once CI has passed 👍

**Pull request merged**